napari/_qt/layer_controls/_tests/test_qt_labels_layer.py

```diff
@@ -115,3 +115,16 @@ def test_preserve_labels_checkbox(make_labels_controls):
     assert not layer.preserve_labels
     qtctrl.preserveLabelsCheckBox.setChecked(True)
     assert layer.preserve_labels
+
+
+def test_change_label_selector_range(make_labels_controls):
+    """Tests that changing the label selector range updates the layer."""
```

Comment:
```suggestion
    """Changing the label layer dtype should updated label selector range."""
```

If I'm reading the test correctly, this doesn't check that updating the selector range updates the layer, but rather the opposite, right?